packages/components/src/components/form/Select/index.tsx

```diff
@@ -133,6 +160,7 @@ const Select = ({
     width,
     variant = 'large',
     noTopLabel = false,
+    minWidth = '50px',
```

Comment:
I am not sure if we want apply some `minWidth` for all selects. Before It was used only for "CleanSelect" variant, "Select" component didn't have such a constraint. I would sleep better if we apply the least amount of custom styles necessary to prevent annoyance in the future 😁